data/dr-adada.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Abdul-Rahim Adada Mohammed",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "An Economist turned **Web Developer/Data Scientist**. Love reading and writing",
+  "avatar": "https://github.com/dr-adada.png",
+  "links": [
```

Comment:
You forgot to add tags, you can add tags like this:
  "tags": ["Data Scientist", "React"],

Reply:
Thanks